### DISCUSSION TITLE
How to edit settings Gradle file？

### DISCUSSION THREAD

**Discussion by blinlincom** [Sketchware-Pro/Sketchware-Pro#479] [Q&A] [answered]
How to edit settings Gradle file？

**Comment by j-px** [comment, marked as answer]
Sketchware and Sketchware Pro don't use Gradle to build projects. Editing any .gradle file (which are all auto-generated by Sketchware Pro) is useless therefore.

**Comment by AdesonBezerra** [comment]
It would be nice to be able to edit the gradle
